.github/workflows/ci-conan.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
           - name: Install packages
             run: |
-                sudo apt-get install -y gcc g++ cmake wget git python3 python3-pip ccache
+                sudo apt-get install -y gcc g++ cmake wget git python3 python3-pip ccache libjemalloc-dev
```

Comment:
Does it really necessary? Conan recipe contains jemalloc

Reply:
Works without it :star_struck: